app/src/main/java/org/mozilla/focus/utils/UrlUtils.java

```diff
@@ -123,7 +123,10 @@ public static boolean isInternalErrorURL(final String url) {
         return "data:text/html;charset=utf-8;base64,".equals(url);
     }
 
-    public static boolean urlsMatchExceptForTrailingSlash(final @NonNull String url1, final @NonNull String url2) {
+    public static boolean urlsMatchExceptForTrailingSlash(final String url1, final String url2) {
+        if (url1 == null || url2 == null) {
```

Comment:
Can you add a comment explaining why we changed this and mention it's hacky? I think we should revert this change when we fix it (hopefully with kotlin's null type safety).